### PR TITLE
Editorial nits from final AD review

### PR DESCRIPTION
**Pull request by kaduk**
I believe that we can just roll these in to the XML file we send to the RFC Editor when they ask for the latest copy after the document is approved.

### CODE REVIEW

**Comment by gselander** [comment]
@kaduk Thanks for proof reading!

Just about the last proposed change, how about replacing "continue to be achieved" with "remain to be fulfilled" or similar? Would sound better in my non-native English hearing ears :-)

**Comment by kaduk** [comment]
"remain to be fulfilled" would sound awkward to me, but "remain satisfied" would work, I think.

File: draft-ietf-ace-oauth-authz.xml [hunk 1]
```diff
@@ -2363,7 +2365,7 @@ Payload:
       the client and the RS in combination with a previously-defined
       CoAP-DTLS profile for interactions between the client and the AS. The
       new profile that combines existing profiles MUST specify how the
-      existing profiles' security properties are achieved. Any profile
+      existing profiles' security properties continue to be achieved. Any profile
```
Comment (kaduk) [inline draft-ietf-ace-oauth-authz.xml:2368]:
```suggestion
      existing profiles' security requirements remain satisfied. Any profile
```

Comment (kaduk) [reply draft-ietf-ace-oauth-authz.xml:2368]: Or maybe
```suggestion
      existing profiles' security properties continue to be provided. Any profile
```

Comment (gselander) [reply draft-ietf-ace-oauth-authz.xml:2368]: Sounds good to me!

**Comment by LudwigSeitz** [comment]
Hi, closing this PR would require a lot of conflict resolution, I'm implementing the changes manually instead. in commit ...

**Pull request closed without merging**